### PR TITLE
[REVIEW] README updates

### PR DESCRIPTION
**Pull request by cjnolet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,71 @@
-# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;RAFT: RAPIDS Analytics Frameworks Toolset</div>
+# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;RAFT: RAPIDS Analytics Frameworks Toolkit</div>
```
Comment (dantegd) [inline README.md:1]: We need to change the word toolset to toolkit here too: https://github.com/rapidsai/raft/blob/c2daa5c97e9dcf6af850f9c04bcbabee914b4500/cpp/CMakeLists.txt#L157

File: README.md [hunk 2]
```diff
@@ -1,6 +1,71 @@
...
+     |
+     |------------ sparse     [sparse matrix / graph primitives]
+     |
+     |------------ spatial    [spatial primitives] (might require SAISS)
+     |
+     |------------ spectral   [spectral clustering]
+     |
+     |------------ stats      [statistics primitives]
+     |
+     |------------ handle.hpp [raft handle]
```
Comment (dantegd) [inline README.md:37]: I think a very minor text somewhere of what is the handle might be good

**Review by dantegd — COMMENTED** [review]
Changes look great! Had a couple of suggestions

File: README.md (hunk 1 above)
Comment (cjnolet) [reply README.md:1]: I'm okay w/ toolset or toolkit, really. For some reason, I personally find toolkit rolls off my tongue more easily. What do you think?

Comment (dantegd) [reply README.md:1]: I think toolkit is slightly nicer too, but I’m fine with either, just mentioning that we need consistency (which can be in another PR to keep this one focused on the README)

**Review by dantegd — APPROVED** [review]

**Review by BradReesWork — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**